### PR TITLE
Updated SixLabors version, increased client timeout for stable tests,…

### PR DESCRIPTION
**Pull request by SoldatXwing**
… improved unit test checks

### CODE REVIEW

File: API.Test/AuthentificationTests.cs [hunk 1]
```diff
@@ -11,16 +11,14 @@ namespace API.Test;
...
-        // Load a file where i saved login data
-        using StreamReader str = new("LoginData.txt");
-        s_Server = str.ReadLine()!;
-        s_LoginName = str.ReadLine()!;
-        s_UserName = str.ReadLine()!;
-        s_Password = str.ReadLine()!;
+        s_Server = "";
+        s_LoginName = "";
+        s_UserName = "";
+        s_Password = ""!;
```
Comment (Suiram1701) [inline API.Test/AuthentificationTests.cs:21]: The "LoginData.txt" file contains log in information to log in into a WebUntis account. The sign in is required for all authentication tests. Why do you want to remove this?

File: WebUntisAPI.Client/WebUntisClient.cs [hunk 2]
```diff
@@ -240,29 +239,36 @@ public async Task<bool> LoginAsync(string server, string loginName, string usern
...
             {
                 _ = LogoutAsync();
                 return false;
             }
 
-            _userType = (UserType)responseObject["result"]["personType"].ToObject<int>();
-            _user = users.FirstOrDefault(user => user.Id == responseObject["result"]["personId"].ToObject<int>());
-
             return true;
+
```
Comment (Suiram1701) [inline WebUntisAPI.Client/WebUntisClient.cs:271]: The whole sign in logic changed. Please take a look at the develop branch.

**Review by Suiram1701 — COMMENTED** [review]
I commented two things that could be a problem. The rest would be fine.

**Comment by Suiram1701** [comment]
@SoldatXwing at first thank you for the PR and it changes. But I have to close this PR without merging it because I worked on the develop branch which is currently 39 commits ahead of the current master branch. If you're still interestet in merging these changes you can create a PR at the develop branch with adjusted changes.

**Comment by SoldatXwing** [comment]
@Suiram1701 Alright, thank you for the review! I’ll take a look at the `develop` branch over the next few days

**Pull request closed without merging**